src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -194,253 +328,349 @@ const CurationPage = (props: ICurationPageProps) => {
+                                        label="FDA Level of Evidence"
+                                        name="propagationLiquidLevel"
+                                        options={[]}
+                                      />
+                                      <RealtimeTextAreaInput
+                                        fieldKey={`mutations/${mutation.firebaseIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/description`}
+                                        inputClass={styles.textarea}
+                                        label="Description of Evidence"
+                                        labelIcon={
+                                          <GeneHistoryTooltip
+                                            historyData={parsedHistoryList}
+                                            location={`${CANCER_TYPE_THERAPY_INDENTIFIER}${getMutationName(mutation)}, ${cancerTypeName}, ${
+                                              treatment.name
+                                            }`}
+                                          />
+                                        }
+                                        name="evidenceDescription"
+                                      />
+                                      <div className="mb-2">
+                                        <AutoParseRefField summary={treatment.description} />
+                                      </div>
+                                    </Collapsible>
+                                  );
+                                })
+                              );
+                            }, [])}
+                          </Collapsible>
+                        );
+                      })}
+                  </Collapsible>
+                </Col>
+              </Row>
+            ))}
           </div>
         )}
         <VusTable hugoSymbol={hugoSymbol} />
+        <Modal isOpen={showFilterModal} toggle={handleToggleFilterModal}>
+          <ModalHeader>
+            <Container>
+              <Row>
+                <Col>Filters</Col>
+              </Row>
+            </Container>
+          </ModalHeader>
+          <ModalBody>
+            <Container>
+              <h6 className="mb-2">Oncogenicity</h6>
+              <Row>
+                {oncogenicityFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setOncogenicityFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
```

Comment:
Can we add the `for` prop and also would need to add and `id` to the Input. I believe it supposed to make it so that when you click on the label, it also checks the box/radio